ui/raidboss/emulator/data/network_log_converter/LineEvent0x18.js

```diff
@@ -32,9 +32,9 @@ class LineEvent0x18 extends LineEvent {
   }
 
   convert(repo) {
-    this.abilityName = repo.Abilities[this.abilityId.toUpperCase()];
+    this.abilityName = LineEvent0x18.showAbilityNamesFor[this.abilityId.toUpperCase()];
```

Comment:
Could you also s/Ability/Effect/ to make it clearer what these ids are?

Reply:
I can rename the `Ability` portions without any issue, but renaming `this.abilityId` will impact `CombatantTracker` here:
https://github.com/valarnin/cactbot/blob/57721fc884569564dc7310f894e4d8e1831602f9/ui/raidboss/emulator/data/CombatantTracker.js#L134-L135

I'm not sure if it makes sense to add `effectId` to the CombatantTracker code, or exclude it altogether since it's probably not accurate and has just been working because it should never theoretically fire against a 0x18 line.

Reply:
I think it will impact `CombatantTracker`, but in a positive manner towards a more correct state.  Currently, `this.abilityId` for this line type is an effect id and not an ability id.  I haven't looked if there's any collisions between effect ids in the 0x18 list and ability ids in the combatant search list, but it seems like a potential risk.

I think it's fine to only use abilities to figure out who combatants are.  Abilities are probably sufficient and it's fine to exclude effects.

Reply:
Should be good now.